concepticondata/references/references.bib

```diff
@@ -3102,3 +3102,78 @@ @misc{Hsiu2015
   doi = {10.5281/zenodo.1127804},
   publisher={Center for Research in Computational Linguistics}
 }
+
+@article{Maciejewski2016,
+  title={{Relative meaning frequencies for 100 homonyms: British eDom norms}},
+  author={Maciejewski, Greg and Klepousniotou, Ekaterini},
+  journal={Journal of Open Psychology Data},
+  volume={4},
+  year={2016},
+  publisher={Ubiquity Press}
+}
+
+@misc{Parks1998,
+  title={Wordsmyth English Dictionary-Thesaurus. [Electronic version]},
+  author={Parks, R and Ray, J and Bland, S},
+  year={1998},
+  publisher={University of Chicago},
+  url = {http://www.wordsmyth.net}
+}
+
+@article{Armstrong2012,
+  title={{eDom: Norming software and relative meaning frequencies for 544 English homonyms}},
+  author={Armstrong, Blair C. and Tokowicz, Natasha and Plaut, David C.},
+  journal={Behavior Research Methods},
+  volume={44},
+  number={4},
+  pages={1015--1027},
+  year={2012},
+  publisher={Springer}
+}
+
+@article{BNC2007,
+  title={{The British National Corpus, Version 3}},
+  author={BNC Consortium},
+  journal={Distributed by Oxford University Computing Services},
+  year={2007}
+}
+
+@book{Fellbaum1998,
```

Comment:
Address is missing. Instead of `2.1`, you should write `version = {2.1}` as an extra field. Also make sure to check that the year is okay. [Evobib](http://digling.org/evobib/?bibtex=WordNet2010) gives as title "WordNet. A lexical database fo English". I suggest to have a look at how people quote this usually. There may even be a paper by Fellbaum, in which she presents the DB, or citation recommendations, etc.

Reply:
I checked the website for the correct citation: https://wordnet.princeton.edu/citing-wordnet

Christiane Fellbaum (1998, ed.) WordNet: An Electronic Lexical Database. Cambridge, MA: MIT Press.

I added the version according to the descriptions of the variables in Maciejewski.